sklearn/naive_bayes.py

```diff
@@ -579,7 +579,9 @@ class MultinomialNB(BaseDiscreteNB):
     ----------
     C.D. Manning, P. Raghavan and H. Schuetze (2008). Introduction to
     Information Retrieval. Cambridge University Press, pp. 234-265.
-    http://nlp.stanford.edu/IR-book/html/htmledition/naive-bayes-text-classification-1.html
+
+    http://nlp.stanford.edu/IR-book/html/htmledition/\
```

Comment:
No, it was not aligned properly when written as at https://github.com/ragv/scikit-learn/blob/0.15.2/sklearn/naive_bayes.py#L414.

See dev docs at http://scikit-learn.org/dev/modules/generated/sklearn.naive_bayes.MultinomialNB.html